### PR TITLE
Document initial RFC process

### PR DESCRIPTION
**Pull request by dcmiddle**
Use process adapted from Rust project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,230 @@
...
+  - [Help this is all too informal!]
+  - [License]
+
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+Ursa or any of its sub-components including but not limited to Ursa
+Core, Ursa Supply Chain, Ursa Seth, the various Ursa SDKs, or the
```
Comment (manudrijvers) [inline README.md]: What are these exactly? Did I miss something?

File: README.md [hunk 2]
```diff
@@ -1 +1,230 @@
...
+
+Furthermore, the fact that a given RFC has been accepted and is "active"
+implies nothing about what priority is assigned to its implementation, nor does
+it imply anything about whether a Ursa developer has been assigned the task of
+implementing the feature. While it is not *necessary* that the author of the
+RFC also write the implementation, it is by far the most effective way to see
+an RFC through to completion: authors should not expect that other project
+developers will take on responsibility for implementing their accepted feature.
+
+Modifications to "active" RFCs can be done in follow-up pull requests. We
```
Comment (manudrijvers) [inline README.md]: Do follow-up PRs undergo the same procedure as a new RFC, or is it simpler to do a follow-up?

**Review by manudrijvers — COMMENTED** [review]
Sounds good to me! Just two small questions.

File: README.md (hunk 1 above)
Comment (dcmiddle) [reply README.md]: oops. cut and paste error. i'll push up a fixup.

File: README.md (hunk 2 above)
Comment (dcmiddle) [reply README.md]: It should be easier to touch up the language. If there's a substantial change to the design then it should be a new RFC. Maybe for small changes to previously approved RFCs we just revert to the existing PR approval, i.e. approval from two different maintainers who are different from the submitter and who ideally represent different viewpoints.

**Review by mikelodder7 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,228 @@
...
+  - Backward incompatible changes
+  - Changes that affect the security of communications or administration
+
+Some changes do not require an RFC:
+
+  - Rephrasing, reorganizing, refactoring, or otherwise "changing shape does
+    not change meaning".
+  - Additions that strictly improve objective, numerical quality criteria
+    (warning removal, speedup, better platform coverage, more parallelism, trap
+    more errors, etc.)
```
Comment (hartm) [inline README.md]: Can we provide some examples here that are crypto-related for guidance?  For instance, proposing a new protocol definitely should require an RFC.  But what about a new implementation of an existing protocol (that could be very different from the old implementation, perhaps with different security properties)?

Comment (dcmiddle) [reply README.md]: Will update with examples including that new implementations of previously approved algorithms require RFCs. All RFCs will include the motivation for the request, i.e. what is the upstream requirement.

**Comment by dcmiddle** [comment]
@hartm added crypto examples and RFC template

File: README.md (hunk 3 above)
Comment (hartm) [reply README.md]: Saw the change.  Thanks!  I hope as the RFCs come in, we can add some more concrete examples.

**Review by hartm — APPROVED** [review]

File: 0000-template.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
+- Feature Name: (fill me in with a unique identifier, my_awesome_feature)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- Ursa Issue: (leave this empty)
+
```
Comment (dhuseby) [inline 0000-template.md]:
```suggestion
- Version: v0.1

```

**Pull request merged**